src/app/shared/shared.module.ts

```diff
@@ -6,10 +6,11 @@ import { EmbedDirective } from './embed.directive';
 import { HighlightModule } from 'ngx-highlightjs';
 import { NgbModule } from '@ng-bootstrap/ng-bootstrap';
 import {MatTabsModule} from '@angular/material/tabs';
+import { MatDialogModule } from '@angular/material/dialog';
 
 @NgModule({
     declarations: [DemoContainerComponent, EmbedDirective],
     exports: [DemoContainerComponent, SvgIconComponent, SpinnerComponent, EnumDisplayPipe, DateDisplayPipe, SubjectDisplayPipe, ResultTableComponent, PhoneNumberPipe, MuziehDateTimeModule, ButtonDirective],
-    imports: [MuziehComponentsModule, CommonModule, HighlightModule, NgbModule, MatTabsModule],
+    imports: [MuziehComponentsModule, CommonModule, HighlightModule, NgbModule, MatTabsModule, MatDialogModule],
```

Comment:
Do we still need this, if we import it into only modal-component and modal-demo-component?

Reply:
good point, i removed it